packages/graphql/tests/tck/issues/2396.test.ts

```diff
@@ -243,19 +231,7 @@ describe("https://github.com/neo4j/graphql/issues/2396", () => {
 
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
             "MATCH (this:\`Mandate\`)
-            WHERE ((this.price >= $param0 AND EXISTS {
-                MATCH (this)-[:HAS_VALUATION]->(this0:\`Valuation\`)
-                WHERE EXISTS {
-                    MATCH (this0)-[:VALUATION_FOR]->(this1:\`Estate\`)
-                    WHERE (this1.area >= $param1 AND this1.floor >= $param2 AND this1.estateType IN $param3 AND EXISTS {
-                        MATCH (this1)-[:HAS_ADDRESS]->(this2:\`Address\`)
-                        WHERE EXISTS {
-                            MATCH (this2)-[:HAS_POSTAL_CODE]->(this3:\`PostalCode\`)
-                            WHERE this3.number IN $param4
-                        }
-                    })
-                }
-            }) AND this.archivedAt IS NULL)
+            WHERE ((this.price >= $param0 AND single(this3 IN [(this)-[:HAS_VALUATION]->(this3:\`Valuation\`) | this3] WHERE single(this0 IN [(this3)-[:VALUATION_FOR]->(this0:\`Estate\`) | this0] WHERE (this0.area >= $param1 AND this0.floor >= $param2 AND this0.estateType IN $param3 AND single(this2 IN [(this0)-[:HAS_ADDRESS]->(this2:\`Address\`) | this2] WHERE single(this1 IN [(this2)-[:HAS_POSTAL_CODE]->(this1:\`PostalCode\`) | this1] WHERE this1.number IN $param4)))))) AND this.archivedAt IS NULL)
```

Comment:
The formatting here is all messed-up

I guess this is caused by going from using the cypher-builder to using raw cypher. Would it be possible to move this into Cypher builder or it would be too out of scope here?